src/window/building/distribution.c

```diff
@@ -1124,3 +1143,112 @@ void window_building_draw_mess_hall(building_info_context *c)
     window_building_draw_employment(c, 308);
     window_building_supplier_draw_foreground(c);
 }
+
+static void window_building_draw_monument_caravanserai_construction_process(building_info_context *c)
+{
+    window_building_draw_monument_construction_process(c, TR_BUILDING_CARAVANSERAI_PHASE_1,
+                                                       TR_BUILDING_CARAVANSERAI_PHASE_1_TEXT, TR_BUILDING_MONUMENT_CONSTRUCTION_DESC);
+}
+
+option_menu_item caravanserai_policy_options[3] = {
```

Comment:
This should be a `static const` and should go right above `static struct { ... } data;`.

Reply:
ok so option_menu_item in culture.c too ?

Reply:
If you didn't change that part of `culture.c`'s code, keep it as is. It can be fixed later.